### PR TITLE
✅ Slike Video Audio Player - AMP Component Introduced 

### PR DESCRIPTION
**Pull request by SlikeTech**
Slike Studio is a TIL's in-house audio-video platform to meet the enterprise needs, which provides live broadcast, on-demand video & podcast solutions. We created this leading Video Platform-as-a-Service, enabling different businesses to create any video solution and workflow, whether standalone or tightly coupled.

This AMP Component is a new component meant for embedding Slike media player, there are certain mandatory parameters as well as customised ones which we can use as per the user configurations required, details of which can be found inside.

Will Yield in better loading time for users as well as the maintaining servers.

### CODE REVIEW

**Comment by SlikeTech** [comment]
@alanorozco Need your help on this **Missing required OWNERS approvals! Suggested**
As per my knowledge I have all the approvals

**Comment by honeybadgerdontcare** [comment]
@SlikeTech I've added @twifkak for reviewing the validator related files. Note any new files should be copyright 2021. Thanks!

**Comment by SlikeTech** [comment]
Copyright modified

File: extensions/amp-slike/validator-amp-slike.protoascii [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#
+
+tags: {  # amp-slike
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-slike"
+    version: "0.1"
+    version: "latest"
+    requires_usage: EXEMPTED
```
Comment (twifkak) [inline extensions/amp-slike/validator-amp-slike.protoascii:24]: I would recommend ERROR here. EXEMPTED is typically reserved for legacy extensions: https://github.com/ampproject/amphtml/blob/91aaf6f3e28c28117fc6ca0eb660d39161e387ee/validator/validator.proto#L496-L498

Alternatively, it is possible to use NONE, if there is some useful non-nil behavior when the `<script>` is on the page but not the `<amp-slike>`, but from a quick glance at the docs & code that doesn't appear to be the case.

File: extensions/amp-slike/0.1/test/validator-amp-slike.out [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+|    <style amp-custom>
+|      amp-slike {
+|        color: red;
+|      }
+|    </style>
+|    <script async custom-element="amp-slike" src="https://cdn.ampproject.org/v0/amp-slike-0.1.js"></script>
+|    <script async src="https://cdn.ampproject.org/v0.js"></script>
+|  </head>
+|  <body>
+|    <amp-slike layout="responsive" width="150" height="80"></amp-slike>
```
Comment (twifkak) [inline extensions/amp-slike/0.1/test/validator-amp-slike.out:19]: This does not appear to match the `.html` file. I would expect to see FAIL up top, with the other two invalid amp-slike instances & their error messages here. Try running `gulp validator --update_tests`.

**Review by twifkak — CHANGES_REQUESTED** [review]
Sorry for the delay! For validation, this looks pretty close; just a couple changes necessary, I think.

File: extensions/amp-slike/validator-amp-slike.protoascii (hunk 1 above)
Comment (twifkak) [reply extensions/amp-slike/validator-amp-slike.protoascii:24]: I've since learned you can just delete this line since ERROR is the default value.

**Comment by SlikeTech** [comment]
@@twifkak changes done.

**Comment by SlikeTech** [comment]
@twifkak changes done.

**Comment by SlikeTech** [comment]
@twifkak requires_usage deleted

**Comment by twifkak** [comment]
Sorry for the delay. The fix to the protoascii looks good to me, thanks!

 `gulp validator --update_tests` still needs to be run (or the changes committed). See the failing test at https://travis-ci.com/github/ampproject/amphtml/jobs/471827545.

**Comment by Gregable** [comment]
@SlikeTech is this PR still something you are working on?

**Comment by alanorozco** [comment]
@SilkeTech If you require followup on this PR, please follow the [contribution guidelines](https://github.com/ampproject/amphtml/blob/main/contributing/contributing-code.md). You need to file an Intent-to-Implement issue and it has to be approved during a design review. Please include in your I2I an explanation why you cannot use `amp-video-iframe` instead, [per  the 3rd party video player guidelines](https://github.com/ampproject/amphtml/blob/main/spec/amp-3p-video.md).

**Comment by Gregable** [comment]
Closing for inactivity.

**Pull request closed without merging**